src/Makefile.am

```diff
@@ -5,8 +5,9 @@ lib_LTLIBRARIES = pkcs11_provider.la
 SHARED_EXT=@SHARED_EXT@
 
 pkcs11_provider_la_SOURCES = \
-	asymmetric_cipher.c \
+asymmetric_cipher.c \
```

Comment:
why this change?

Reply:
> Boy is it ugly to support multiple platforms ... I wonder if this is the end of it, do we have specific uses in mind though?

It compiles fine on macOS now.

> That said, please move the file under a directory named "isc" like with the "oasis" headers.

Why? There's nothing ISC specific, and I (personally) wrote the header, so it's ok to have it as public domain or whatever. There's no magic in there.

Reply:
> why this change?

This was unintentional. I do have some white space formatting macros in my emacs, so perhaps it reformatted the whitespace. I'll revert this.